### DISCUSSION TITLE
Welcome to WinGetty Discussions!

### DISCUSSION THREAD

**Discussion by thilojaeggi** [thilojaeggi/WinGetty#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by xrbh70** [comment]
How can I configure winget to allow tcp port on the Downloading url?
The source has the  has the Company https://internal_server.net:8082/wg/ for private repository    TCP port 8082
But
It downloads from Downloading https://diam-devops.nadenso.net/api/download/Python.Python114/3.11.4/x64/both without the tcp port
